bundles/net.tourbook/src/net/tourbook/ui/views/tourDataEditor/TourDataEditorView.java

```diff
@@ -9032,6 +9182,7 @@ private void updateUI_Tab_1_Tour() {
        */
       _tourContainer.layout(true);
 
+      enableControls();
```

Comment:
`enableControls()` is already called after this method, when there is a need to call `enableControls()`, then it should be called outside of `updateUI_Tab_1_Tour()`

![image](https://user-images.githubusercontent.com/1283445/157382283-5ff378de-1d36-4539-85d4-0aa7f2b602f3.png)

Reply:
Indeed, I didn't see that. Removed.